src/Apps/Collect/Components/Collection/CollectionFilterContainer.tsx

```diff
@@ -29,6 +29,7 @@ export const CollectionFilterContainer: React.FC<
       user={user}
       mediums={mediumAggregation.counts as any}
       timePeriods={timePeriodAggregation.counts as any}
+      isCollection
```

Comment:
Not required for this PR but since we want to eventually enable price sorting for all artwork grids, we could explore a more generic API for enabling/disabling filter options. For instance:

```typescript
enum CollectSortingOptions {
  default = "default"
  price = "price"
  year = "year"
  ....
}

<FilterContainer
  sortOptions={["price", "year"]}
/>
```

Reply:
Hey man take a look at the latest iteration. I had some downtime waiting for gravity to go out to staging, so I tried to clean this up. I took it a little further than this - passing a large array of sort options around through multiple levels of props feels weird. So instead I created named sets of options, and we now pass the name through the props chain.

This feels a lot cleaner?